README.md

```diff
@@ -15,6 +15,10 @@ a hardware & software project that lets people attending the [Recurse Center](ht
 - the oauth/web/UI server is [here](https://github.com/gregsadetsky/checkintopus)
 - the hardware raspberry pi code is in this repo
 
+## Production environment
+We use a raspberry pi located in the hub's wifi
```

Comment:
located is probably a better word for a physical location. rephrase it

Reply:
What do you think about the currently committed sentence?
`We use a raspberry pi that is located at the hub and connected to the network via wifi`